openedx/features/course_experience/tests/views/test_course_home.py

```diff
@@ -141,6 +141,29 @@ def setUp(self):
         # Add a welcome message
         create_course_update(self.course, self.staff_user, TEST_WELCOME_MESSAGE)
 
+    def _create_test_user(self, user_type):
```

Comment:
This got accidentally picked up in the rebase. I moved this logic into a different place.